docs/index.rst

```diff
@@ -0,0 +1,299 @@
+ADB Enhanced |Downloads| |PyPI version| |Build Status| |CircleCI|
+=================================================================
+
+.. figure:: docs/logo.png
+   :alt: Logo
+
+   Logo
+
+Swiss-army knife for Android testing and development. A command-line
+interface to trigger various scenarios like screen rotation, battery
+saver mode, data saver mode, doze mode, permission grant/revocation.
+
+Release announcement
+--------------------
+
+See `Release
+announcement <https://ashishb.net/tech/introducing-adb-enhanced-a-swiss-army-knife-for-android-development/>`__
+
+Installation
+------------
+
+``sudo pip3 install adb-enhanced``
```

Comment:
Bad idea on many systems, as it can mess up the system python. I'd mention using a virtualenv or `--user`.

Reply:
Hi @m3nu ,

How can one add `adbe` command to path without `sudo pip3 install adb-enhanced`?

Reply:
Users can edit their `$PATH` or they will know to use `sudo` if appropriate.

Another alternative is to offer a single fat binary, as is common with Go projects.

Sadly I have to admit that many projects show the `sudo pip` way.  So you won't be in bad company if you stick with it.

Reply:
> Sadly I have to admit that many projects show the `sudo pip` way. So you won't be in bad company if you stick with it.

Thanks. I will stay with `sudo pip3` for now and later look into the packaged binary approach.